incubator/command-generate/src/main/java/io/aklivity/zilla/runtime/command/generate/internal/asyncapi/mqtt/proxy/AsyncApiMqttProxyConfigGenerator.java

```diff
@@ -258,9 +242,7 @@ private BindingConfigBuilder<NamespaceConfigBuilder<NamespaceConfig>> injectMqtt
         {
             String topic = channelEntry.getValue().address.replaceAll("\\{[^}]+\\}", "*");
             Map<String, Message> messages = channelEntry.getValue().messages;
-            Message firstMessage = messages.entrySet().stream().findFirst().get().getValue();
-            String contentType = MessageView.of(asyncApi.components.messages, firstMessage).contentType();
-            if (APPLICATION_JSON.equals(contentType))
+            if (APPLICATION_JSON.equals(resolveContentType()))
```

Comment:
I think we need a pattern matcher here to cover all the possibilities for `json` messages, as there are many more than just `application/json`.

Reply:
done, pls chk